code/modules/jobs/job_types/chaplain.dm

```diff
@@ -74,57 +77,29 @@
 		new_bible = player_client.prefs.custom_names["bible"]
 
 	switch(lowertext(new_religion))
-		if("christianity") // DEFAULT_RELIGION
-			new_bible = pick("The Holy Bible","The Dead Sea Scrolls")
-		if("buddhism")
-			new_bible = "The Sutras"
-		if("clownism","honkmother","honk","honkism","comedy")
-			new_bible = pick("The Holy Joke Book", "Just a Prank", "Hymns to the Honkmother")
-		if("chaos")
-			new_bible = "The Book of Lorgar"
-		if("cthulhu")
-			new_bible = "The Necronomicon"
-		if("hinduism")
-			new_bible = "The Vedas"
-		if("homosexuality")
-			new_bible = pick("Guys Gone Wild","Coming Out of The Closet")
-		if("imperium")
-			new_bible = "Uplifting Primer"
-		if("islam")
-			new_bible = "Quran"
-		if("judaism")
-			new_bible = "The Torah"
-		if("lampism")
-			new_bible = "Fluorescent Incandescence"
-		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "meme", "memes")
-			new_bible = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition","F.A.T.A.L. Rulebook")
+		if("homosexuality", "gay", "penis", "ass", "cock", "cocks")
+			new_bible = pick("Guys Gone Wild","Coming Out of The Closet","War of Cocks")
+			switch(new_bible)
+				if("War of Cocks")
+					B.deity_name = pick("Dick Powers", "King Cock")
+				else
+					B.deity_name = pick("Gay Space Jesus", "Gandalf", "Dumbledore")
```

Comment:
What the fuck lmao

Reply:
https://github.com/tgstation/tgstation/blob/master/code/modules/jobs/job_types/chaplain.dm

Reply:
Hahahaha